### ISSUE TITLE
Feature Request: save export settings as file

### ISSUE THREAD

**Issue by joelnitta** [retorquere/zotero-better-bibtex#2221]
Sorry if this is already documented somewhere, but I didn't see anything...

For the purposes of a collaborative project, it would be helpful to be able to save the export settings for an automatic export as a file. The resulting bib (yaml, etc) file from an automatic export can have various formats depending on what options were used when exporting. It may be difficult to convey to a collaborator exactly what options to click on/off etc to get the same results. If we could just import a file with all of the settings included it would make that much easier.

For example, I needed to provide a long list of settings here: https://github.com/fernphy/ferncal/pull/8#issue-1314053326

**Comment by joelnitta** [comment]
Sorry for not posting the log, here it is: HIHJBF3G-apse

**Comment by qqobb** [comment]
@joelnitta: I haven't tried this yet, but there's now an "Import BetterBibTeX preferences..." button in BBT's "Import" preference tab. You probably need to first export some item with the "BetterBibTeX JSON" translator, which will give you a file that includes all settings, see #1942.

@retorquere: An analogous "Export BetterBibTeX preferences..." button in BBT's "Export" preference tab would be nice, perhaps in the "Miscellaneous" subtab.  This should be more intuitive. When I saw the "Import prefs" button, I looked for an "Export prefs" button, but couldn't find one. I guess it should only export the "config/preferences" data and perhaps "version". Maybe this is already planned, as it's mentioned in the other issue.
